testing/run_unit_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This should be called from one of the CI test scripts (travis) -- I'd recommend testing/test_modules.sh -- even though it's not modules (will rename later).

Reply:
There already exists a test_modules.sh which is called. test_units?

Reply:
it should go in test_modules... there's a high overhead for a new top-level test, so shouldn't create new ones unless absolutely necessary.